pkg/kubectl-argo-rollouts/info/rollout_info.go

```diff
@@ -69,8 +69,8 @@ func NewRolloutInfo(
 			roInfo.ActualWeight = "100"
 		} else if ro.Status.AvailableReplicas > 0 {
 			for _, rs := range roInfo.ReplicaSets {
-				if rs.Canary {
-					roInfo.ActualWeight = fmt.Sprintf("%d", (rs.Available*100)/ro.Status.AvailableReplicas)
+				if rs.Canary && rs.Status == "Healthy" {
```

Comment:
I think we are going to need to have 2 cases here: one for traffic management and one without. 

The old code covers the non-traffic management case, where the traffic is split evenly amount all the available replicas. Since no tools are used to split the traffic split, the actual percentage of traffic going to the new version is the number of available replicas in the new ReplicSet over the total number of replicas in all the ReplicaSets of that Rollout.

In the traffic management case, we should only use SetWeight info. The traffic shaping tools will ensure the percentage instead of relying on replicas count. You can detect the trafficManagement by checking that `.spec.strategy.canary.trafficManagement != nil`.

Reply:
@dthomson25 Thank you for the review! Let me fix it quickly 👍

Reply:
@dthomson25 I've fixed it, so would you mind reviewing the change?